### PR TITLE
Libreelec 7.0

### PR DESCRIPTION
**Pull request by regiogids**

### CODE REVIEW

**Comment by lrusak** [comment]
Is this a mistake?

File: packages/sysutils/busybox/scripts/fs-resize [hunk 1]
```diff
@@ -39,19 +39,20 @@ if [ -e /storage/.please_resize_me ] ; then
   sync
 
   # just in case
-  if [ ! -z "$DISK" -a ! -z "$PART" ] ; then
-    # get storage partition start
-    PART_START=$(parted -s -m $DISK unit b print |grep -v ^/dev |grep -v BYT | grep ^2: |  cut -f2 -d ":")
-
-    # failed to get partition start offset ?
-    if [ ! -z "$PART_START" ] ; then
+  if [ ! -z "$PART_START" ] ; then
```
Comment (vpeter4) [inline packages/sysutils/busybox/scripts/fs-resize]: Value of PART_START is 0 and no resize is made.

And btw: LibreELEC 7.0 will not be updated anymore. Only addons.

**Comment by vpeter4** [comment]
Out of curiosity: what improvements this changes do?

**Comment by chewitt** [comment]
I'm closing the PR because these this branch is closed to development. If you'd like to explain what you're trying to achieve with these changes we can advise whether they would be accepted to an open branch like 8.0 or master (9.0). Thanks.

**Pull request closed without merging**